src/project/project.service.ts

```diff
@@ -23,6 +24,23 @@ import {
 } from './dtos/project.show.dto';
 import { Project } from './project.entity';
 import { QueryCreator } from './project.query.creator';
+import {
+  Enrollment,
+  ProjectRole,
+  RequestState,
+} from '../enrollment/enrollment.entity';
+import { EnrollmentRequestDto } from '../enrollment/dtos/enrollment.request.dto';
+import { UnenrollDto } from '../enrollment/dtos/unenroll.dto';
+import {
+  EnrollmentRequestShowDto,
+  EnrollmentRequestsShowDto,
+} from '../enrollment/dtos/enrollment-request.show.dto';
+import { EnrollmentRequestAdminDto as EnrollmentRequestAdminDto } from '../enrollment/dtos/enrollment-request-admin.dto';
+import { EnrollmentChangeRole } from '../enrollment/dtos/enrollment-change-role';
+
+const projectNotFoundError = new NotFound(
```

Comment:
esta const la usamos? 
Porque veo que abajo haces un new NotFound(...) cada vez con el mismo texto que la const en vez de usarla

Reply:
Fixed in https://github.com/team120/Universiteams/commit/fee413f5b3a4d8e537985e45a0094af936e18f3c